pkg/expect/expect.go

```diff
@@ -189,3 +192,9 @@ func (ep *ExpectProcess) ProcessError() error {
 	}
 	return ep.err
 }
+
+func (ep *ExpectProcess) Lines() []string {
+	ep.mu.Lock()
+	defer ep.mu.Unlock()
+	return ep.lines
```

Comment:
It's just shadow copy, please make sure there is no any race-condition.

Reply:
This is a backport, I try to fix it on main first